src/pydata_sphinx_theme/assets/styles/index.scss

```diff
@@ -186,6 +188,11 @@ footer {
   &.no-sidebar {
     border-right: 0;
   }
+
+  .sidebar-end-items {
+    margin-top: auto;
+    margin-bottom: 0.5em;
```

Comment:
Is this 0.5em compatible with other objects' margins?
Looking at the codebase, I do not see a lot of 0.5em references which leads me to think there is not a general pattern, but I want to make sure that is the case...

Reply:
you're right, not a general pattern. I think here I was trying to apply a similar behavior to the one that we added in https://github.com/pydata/pydata-sphinx-theme/pull/541 - can you think of a better way to do this 🤔

Reply:
I think this is an OK-ish approach if there is not already a generalization. Maybe you want to raise it to 1.0em to be in sync with #541?